ios/sbm/AppDelegate.mm

```diff
@@ -1,42 +1,47 @@
 #import "AppDelegate.h"
-#import <UserNotifications/UserNotifications.h>
-#import <RNCPushNotificationIOS.h>
-
 #import <React/RCTBundleURLProvider.h>
-
 #import <Firebase.h>
 #import "RCTLinkingManager.h"
 #import "RNSplashScreen.h"
+#import <UserNotifications/UserNotifications.h>
+#import <RNCPushNotificationIOS.h>
+#import "SendbirdNotificationHelper.h"
 
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   [FIRApp configure];
-  
-  self.moduleName = @"notifications";
+
+  self.moduleName = @"sendbirdBusinessMessaging";
   // You can add your custom initial props in the dictionary below.
   // They will be passed down to the ViewController used by React Native.
   self.initialProps = @{};
   
-  // Define UNUserNotificationCenter
   UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
   center.delegate = self;
   
+  [SendbirdNotificationHelper setAppGroup:@"group.sample.sbm.react-native"];
```

Comment:
You can add App Groups to `notifications` target

`Signing & Capabilities > TARGETS > notifications > +Capability > App Groups`

![image](https://github.com/sendbird/sbm-react-native-sample/assets/26326015/15fa4d2e-2077-4843-af7d-44b712e3b2c2)